VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participant-waiting-room/participant-waiting-room.component.ts

```diff
@@ -113,8 +112,8 @@ export class ParticipantWaitingRoomComponent extends WaitingRoomBaseComponent im
         }
     }
 
-    announceHearingIsAboutToStart(): void {
-        this.notificationSoundsService.playHearingAlertSound();
+    async announceHearingIsAboutToStart(): Promise<void> {
```

Comment:
i cant comment on code you havnt changed yet :P
could we use executeWaitingRoomCleanup ?

Reply:
in ngOnDestroy

Reply:
we can! done!